beamer/health/notification_state.py

```diff
@@ -0,0 +1,34 @@
+import json
+import os
+from pathlib import Path
+
+
+class NotificationState:
```

Comment:
I think we can just move everything from this file to `beamer.health.notify` and remove this module. It's related to notifications anyway.